src/main/java/com/openosrs/injector/injectors/raw/RasterizerAlpha.java

```diff
@@ -125,27 +125,39 @@ public void inject() throws Injexception
 						StackContext alphaPop = isub.getPops().get(0);
 						InstructionContext alphaPusher = alphaPop.getPushed();
 						InstructionContext isubResult = isub.getPushes().get(0).getPopped().get(0);
+						boolean shouldSubtractLocal = false;
 
 						if (pushesToSameVar(isubResult, alphaPusher))
```

Comment:
is pushesToSameVar side effect free? if so, comment that out as well

Reply:
nvm

Reply:
ok